config/application.rb

```diff
@@ -10,6 +10,7 @@
 require 'action_mailer/railtie'
 # require "action_view/railtie"
 # require "sprockets/railtie"
+require_relative '../lib/statsd_middleware'
```

Comment:
`require` doesn't work, and I think it's because of where this file gets loaded in the bootstrap process. I'm guessing the `lib` directory hasn't been added to the load path by Rails yet.